embuilder.py

```diff
@@ -160,6 +160,8 @@ def build_port(port_name, lib_name, params):
       build_port('sdl2', 'libsdl2.bc', ['-s', 'USE_SDL=2'])
     elif what == 'sdl2-image':
       build_port('sdl2-image', 'libsdl2_image.bc', ['-s', 'USE_SDL=2', '-s', 'USE_SDL_IMAGE=2'])
+    elif what == 'sdl2-net':
+      build_port('sdl2-net', 'libsdl2_net.bc', ['-s', 'USE_SDL=2', '-s', 'USE_SDL_NET=2'])
```

Comment:
Further up in this file is a `tasks = ['libc', 'libc-mt',` line which represents all tasks - can you add sdl2-net to that? (it's also missing sdl2-ttf if you feel like adding that too, or I'll do it in a later PR).
Additionally, the big help text at the top of this file also needs sdl2-net adding.